### PR TITLE
First version of the ioSPI notebook tutorial

### PR DESCRIPTION
**Pull request by luis-sribeiro**

### CODE REVIEW

File: notebooks/tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,431 @@
+{
```
Comment (fredericpoitevin) [inline notebooks/tutorial.ipynb:1]: <p>Are we expecting something to be printed here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/compSPI/ioSPI/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (fredericpoitevin) [inline notebooks/tutorial.ipynb:1]: <p>Nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/compSPI/ioSPI/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Comment by geoffwoollard** [comment]
Luis, I suggest you 
1. Make a repo of the main compSPI fork locally
2. Make an empty feature branch on the main compSPI fork, off master. It will be just an empty feature branch. Let's call this `compSPI:luis_notebook`
3. Do a PR of your new feature branch from your fork ([luis-sribeiro:master] for PR https://github.com/compSPI/ioSPI/pull/52) to `compSPI:luis_notebook`
4. Now the feature is within the compSPI organization, and the authentication will work differently and the PR won't fail.
5. Now do a PR of `compSPI:luis_notebook` --> `compSPI:master`

In the future you can avoid this by working on the compSPI organization repo, instead of your own fork.

**Comment by geoffwoollard** [comment]
@luis-sribeiro I confirm that the above steps all worked for me, including this PR that passed: https://github.com/compSPI/ioSPI/pull/61

**Comment by fredericpoitevin** [comment]
I created the `luis_master` branch on the main fork and tests pass there: #64 

@luis-sribeiro I am going to close this pull request - could you please address the comments in the other one (you can push to `origin luis_master` to update it)

**Pull request closed without merging**